### PR TITLE
Archive ESPP submission as an HSF note

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
This is the LaTeX for the submitted HSF contribution to the European Particle Physics Strategy Update 2018-2020.

As it's not a technical note I created a new generic note category and used the usual YEAR-NN scheme to number things (this is `2018-01`).

### CODE REVIEW

**Comment by graeme-a-stewart** [comment]
I have one question, which is should we put the HSF document number, i.e. `HSF-NOTE-2018-01` on this version? It doesn't have it right now as that seemed inappropriate for the actual submission to the ESPP.

File: HSF-Notes/2018-01/espp-hsf-submission.tex [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+\abstract{In 2017 the experimental High-Energy Physics community wrote a
+\emph{Roadmap for HEP Software and Computing R\&D for the
+2020s}\footnote{\href{https://arxiv.org/abs/1712.06982}{\emph{arXiv:1712.06982}}}.
+This effort was organised by the HEP Software
+Foundation\footnote{\href{https://hepsoftwarefoundation.org/}{\emph{https://hepsoftwarefoundation.org/}}}
+(HSF) and was supported by more than 300 physicists from more than 100
+institutes worldwide. It delivered a strategy outlining the most
+important areas in which investment is needed to ensure the success of
+our experimental programme. This contribution to the ESPP is an
```
Comment (eduardo-rodrigues) [inline HSF-Notes/2018-01/espp-hsf-submission.tex]: Apologies for being a pain - we could spell out ESPP in the abstract at least.

File: HSF-Notes/2018-01/espp-hsf-submission.tex [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+lifetimes for experiments, meaning we need long lifetimes for software
+and long-term support for those who develop it. It is thus crucial that
+we offer well defined career paths, including new roles such as Research
+Software Engineers, for those who chose to work in these software areas,
+so as to retain the necessary talent for our discipline. This is
+especially true for those who chose to focus on the software engineering
+of scientific code, where long-term career prospects must be viable.
+This requires endorsing new initiatives to attract funding for a
+sustainable and cooperative scientific software ecosystem at European
+universities and labs.
```
Comment (eduardo-rodrigues) [inline HSF-Notes/2018-01/espp-hsf-submission.tex]:
```suggestion
universities and laboratories.
```

**Comment by eduardo-rodrigues** [comment]
BTW, why not add the PDF of the document in the archive? We do that for TNs for example, and it's helpful to anyone browsing.

File: HSF-Notes/2018-01/espp-hsf-submission.tex (hunk 1 above)
Comment (graeme-a-stewart) [reply HSF-Notes/2018-01/espp-hsf-submission.tex]: Well, the problem is now that we made the submission, so I don't want the archived copy of the document to differ from what we actually gave in to the process.

File: HSF-Notes/2018-01/espp-hsf-submission.tex (hunk 2 above)
Comment (graeme-a-stewart) [reply HSF-Notes/2018-01/espp-hsf-submission.tex]: Ditto.

**Comment by graeme-a-stewart** [comment]
Thanks for the suggestions, @eduardo-rodrigues. I renamed the directory to `HSF-DOC` and also included a copy of the generated PDF, as per TNs.

Following the TN procedure I will also upload this document to Zenodo and generate the DOI.

**Comment by eduardo-rodrigues** [comment]
Ah, fair point, @graeme-a-stewart. Better to be 100% in sync, yes.

Thanks.

**Comment by eduardo-rodrigues** [comment]
I will not merge this just now since this document is a new type and it makes sense for a second person to comment, OK? @jouvin and/or @peremato, happy too? All good IMO.

**Comment by graeme-a-stewart** [comment]
Yes, I agree - someone else should check :-)

I have uploaded the document to Zenodo and added the DOI (thanks for the very clear instructions on how to do that, it was my first time).

**Comment by eduardo-rodrigues** [comment]
Two final points:
- Replace the PDF with the one containing the DOI in it (maybe already the case - did not check).
- On Zenodo, tag the document as HSF (our usual curation :-))

Thanks a lot.

**Comment by graeme-a-stewart** [comment]
Hi @eduardo-rodrigues - both should be ok. The updated PDF was in the last commit and I added the document to our community when I submitted it. (And it needs to be accepted by the curator - which is you, right?)

**Comment by eduardo-rodrigues** [comment]
All HSF editorial members are the curators. This is working, cf. a couple of emails I had sent to the corresponding list. In fact it would be great if somebody could cross-check all is working - works just fine for me.

**Comment by eduardo-rodrigues** [comment]
Otherwise I'm happy to curate, of course ;-).

**Comment by eduardo-rodrigues** [comment]
Can you  @jouvin and/or @peremato have a quick look? Just so that we tick this off before the break :-). Thanks.

**Comment by eduardo-rodrigues** [comment]
Hi @davidlange6, can you have a look at this PR? Thanks a lot.

**Pull request merged**